### PR TITLE
Meeting notes, July 26

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: ember-cli/2018-07/july-26.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2018-07-26
+
+## Attendees
+
+## Agenda
```
Comment (rwjblue) [inline ember-cli/2018-07/july-26.md]: Enter items here...

Comment (rwjblue) [reply ember-cli/2018-07/july-26.md]: * broccoli 2
* ember-template-lint
* jQuery via @ember/optional-features
* travis stages
* eslint during build

Comment (twokul) [reply ember-cli/2018-07/july-26.md]: https://github.com/ember-cli/ember-cli/pull/7941

Comment (twokul) [reply ember-cli/2018-07/july-26.md]: also, discuss dep collisions during the build time

Comment (kellyselden) [reply ember-cli/2018-07/july-26.md]: https://github.com/ember-cli/ember-cli/pull/7935

**Comment by twokul** [comment]
@rwjblue @kellyselden @kategengler

File: ember-cli/2018-07/july-26.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Discussion Notes
+
++ move forward with landing broccoli 2.0 in 3.4-beta cycle (our first LTS)
++ land changes in linter for angle bracket components (support for ember 3.4)
++ backport fixes for `optional-features` to beta branch
++ change `travis.yml` file (for add-ons) to use stages since it's no longer a beta feature (it
+  will increase the job time, but overall time of CI will decrease); it's not
+  that big of a change to have an RFC for it.
++ move eslint to build time; requires an RFC; generating extra modules (for
```
Comment (kellyselden) [inline ember-cli/2018-07/july-26.md]: you mean move eslint out of build?

**Pull request merged**